bootstrap/kubeadm/api/v1alpha4/kubeadmbootstrapconfig_types_test.go

```diff
@@ -27,6 +27,11 @@ import (
 // These tests are written in BDD-style using Ginkgo framework. Refer to
 // http://onsi.github.io/ginkgo to learn more.
 
+var (
+	msDos     = "MS-DOS"
+	trueValue = true
+)
```

Comment:
might be I'm wrong, but those variable seems not being used anywhere

Reply:
It is used in tests:
```
0 ✓ (185ms) 15:15:49 invidian@dellxps15mateusz ~/data/workspaces/clusterapi-flatcar/cluster-api (invidian/ignition-support-pr)$ git grep trueValue
bootstrap/kubeadm/api/v1alpha4/kubeadmbootstrapconfig_types_test.go:    trueValue = true
bootstrap/kubeadm/api/v1alpha4/kubeadmbootstrapconfig_types_test.go:                                                    Inactive: &trueValue,
bootstrap/kubeadm/internal/ignition/clc/clc_test.go:            trueValue := true
bootstrap/kubeadm/internal/ignition/clc/clc_test.go:                            Enabled: &trueValue,
bootstrap/kubeadm/internal/ignition/clc/clc_test.go:                                            Overwrite: &trueValue,
0 ✓ (1.274s) 15:15:52 invidian@dellxps15mateusz ~/data/workspaces/clusterapi-flatcar/cluster-api (invidian/ignition-support-pr)$ git grep msDos
bootstrap/kubeadm/api/v1alpha4/kubeadmbootstrapconfig_types_test.go:    msDos     = "MS-DOS"
bootstrap/kubeadm/api/v1alpha4/kubeadmbootstrapconfig_types_test.go:                                                            TableType: &msDos,
```

Global variable to be able to take a reference of it. I can change it to helper functions as well.

Reply:
thanks for double checking